### PR TITLE
Mount home and cache storage items

### LINKED ISSUE #20: Make research drive available on Jupyter server
The experiments (cylc) and forecast server use rclone and cron jobs to sync data to/from researchdrive.

Inside a notebook it would be nice to access this forecast data aswell.

### LINKED ISSUE #89: Bigger home and scratch dir
In the SRC VM the OS disk almost full need more room for users. Lets have a scratch and home as additional SRC storage items.
The storage items should be made available inside VM as /home and /scratch.

Each storage item can only be mounted on a single VM, so each VM will need own storage items. 

Each storage item is persistent so if reused in a new VM it will contain stuff from old usage. So we should have instructions that explains that home should be reused and scratch should be empty each time.

To keep scratch clean we could remove files older than 14 days.

To replace /home from root disk with a storage item, we will need to 
1. copy /home mounted on root disk to temp home, 
2. perform mount, 
3. copy temp home to new home and 
4. remove temp home

See #77

### LINKED ISSUE #97: "Wflow / Rhine / 2002-01-01 - 2002-12-31 / ERA5"
Similar to #94 got `Exception calling application: /home/sverhoeven/c5663439-c934-4979-9aa9-f3e68a809bcb/ewatercycle_output/wflow_20210903_114726/inmaps/wflow_ERA5_Rhine_1991_2001.nc not found!`

### PR DESCRIPTION
**Pull request by sverhoeven**
Mounts SRC home storage item as /home.

Fixes #89 
Fixes #97
Fixes #20

This PR also 
* switches to systemd spawner for access to log of a users Jupyter servers.
* uses release build of Terria, for smaller js and quicker loading.

### CODE REVIEW

**Comment by sverhoeven** [comment]
Running merrimack 5min pcrglobwb and rhine 5min notebooks on a vagrant box with `rclone_max_gsize: 45` did not give any `no space left` errors.

**Comment by sverhoeven** [comment]
Running wflow/rhine and pcrglobwb/rhine and merrimack at same time. Cache disk was 100% full some times, however the rclone log did not have a `no space left` error. 

So I think we should lower rclone_max_gsize to have some buffer.

**Comment by Peter9192** [comment]
Local build fails for me with 


```sh
TASK [common : Download pip installer] *****************************************
fatal: [default]: FAILED! => {"changed": true, "checksum_dest": null, "checksum_src": "493529f277047b086815c1cdc0cff418cf62870a", "dest": "/opt/get-pip.py", "elapsed": 10, "msg": "The checksum for /opt/get-pip.py did not match fa6f3fb93cce234cd4e8dd2beb54a51ab9c247653b52855a48dd44e6b21ff28b; it was 01249aa3e58ffb3e1686b7141b4e9aac4d398ef4ac3012ed9dff8dd9f685ffe0.", "src": "/home/vagrant/.ansible/tmp/ansible-tmp-1632896035.1524048-30647508401664/tmp2zmux7n0", "url": "https://bootstrap.pypa.io/get-pip.py"}
```

**Comment by sverhoeven** [comment]
I updated the checksum. Can you try again?

File: roles/storage/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+# defaults file for storage
+# Mount point for home directories
+home_location: /home
+# Mount point for alternative home directories, used for syncing old home and as bind mount source
+alt_home_location: /data/volume_3
```
Comment (Peter9192) [inline roles/storage/defaults/main.yml:6]: This comment is a bit unclear to me. Can we explain better what the relation between home and alt_home is?

**Review by Peter9192 — APPROVED** [review]
Again a collection of on-topic and ad-hoc changes 😎 
Local build works fine and looks good (except for one future-timeout error). 

/data/volume_2 is protected, /data/volume_3/vagrant is the same as $HOME (/home/vagrant). 

I find the term "alt_home" to be a bit confusing. The point is that the default VM home disk is a bit small, so instead we mount a separate storage item and use that as the default home, right?

File: roles/storage/defaults/main.yml (hunk 1 above)
Comment (Peter9192) [reply roles/storage/defaults/main.yml]: Thanks, this is great!

**Comment by sverhoeven** [comment]
Was the timeout because it took more then 5 minutes to download a container image?

Yep the root disk is too small. So I move the existing home dirs to a new disk and then mount the disk over the existing home. 

The /data/volume_3/vagrant is the same /home/vagrant are the same because of a bind mount which is like a sym-link for mount points.

**Pull request merged**